docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+---
+
+<br>
+<br>
+
+## 1. Streaming Data From AWS RDS PostgreSQL DB to PostgreSQL DB:
+
+---
+### 1.1 Topology
+
+![m'lady](./images/debezium-postgres.jpg)
+<br>
+---
+
+<br>
+<br>
+
+### 1.2 AWS RDS PostgreSQL DB Configuration
+
+1. Create a new "Parameter Group" at AWS RDS ***Parameter groups*** section and give it a name , for example, "postgres-db-parameter-groups"
+
+2. Be sure to set the new ***Parameter Group*** parameters as follows:
+    
+![m'lady](./images/AWS-RDS-Postgres-DB-Parameter-Group.png) 
+	
+	
+----
+	rds.logical_replication = 1
+    shared_preload_libraries = pg_stat_statements,pg_hint_plan,pgaudit
+    ssl=0
+----
+
+* 3 - Create a new AWS RDS PostgreSQL DB called testdbserver. Set the user name as ***testdbuser***. 
+Give permission to Port Number ***TCP:5432*** to all sources (0.0.0.0/0) via ***DB Security Group***.
+
+![m'lady](./images/AWS-Security-Group-permission.png) 
+
+4. Connect to the database via psql 
+
+----
+    [ec2-user@hostname ~] $ psql -h testdbserver.cuwbhrbzgqk9.us-east-1.rds.amazonaws.com -d testdb -U testdbuser -p 5432
+----
+
+5. Type \d to see a screen like that and be sure that  your user testdbuser is a member of rds_superuser and rds_superuser has ***rds_replication*** right
+
+    `testdb=> \du`
+    
+	![m'lady](./images/Replication-RDS.png) 
+	
+
+6. Display if any replication slots are available
+
+    `testdb=> select * from pg_replication_slots;`
+
+7. Create a replication plugin called ***wal2json*** and name it ***wal2json_rds***
+
+    `testdb=> SELECT * FROM pg_create_logical_replication_slot('wal2json_rds', 'wal2json');`
+
+8. If you wanna delete the slot type this
+    
+    `testdb=> SELECT pg_drop_replication_slot('wal2json_rds');`
+
+9. Create a table named ***customers***
+
+    `testdb=> CREATE TABLE customers (id SERIAL,  first_name VARCHAR(255) NOT NULL,  last_name VARCHAR(255) NOT NULL,  email VARCHAR(255) NOT NULL,  PRIMARY KEY(id));`
+
+10. Change table so that it can ***support replication***
+
+    `testdb=> ALTER TABLE customers REPLICA IDENTITY FULL;`
+
+11. ***Insert some records*** into the source table
+
+----
+    testdb=> insert into customers values(default, 'fatih1', 'kose1', 'fatih1.kose@example.com');
+    testdb=> insert into customers values(default, 'fatih2', 'kose2', 'fatih1.kose@example.com');
+----
+
+### 1.3 Prepare Debezium Test Environment And Complete Configuration On Linux
+
+1. Create a directory under user home directory 
+
+    `[ec2-user@hostname ~] $ mkdir ~/debezium`
+
+2. Download the sample config scripts from ***https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts*** and copy them into that directory
+
+    `[ec2-user@hostname ~] $ cp *.zip debezium/`
+
+3. Enter into that directory
+
+    `[ec2-user@hostname ~] $ cd debezium`
+
+4. ***export DEBEZIUM_VERSION*** environment variable
+
+    `[ec2-user@hostname ~] $ export DEBEZIUM_VERSION=0.7.1`
+
+5. ***Build docker-compose files*** named docker-compose.yaml (default name)
```

Comment:
It'd be nice to show the contents of the Compose file (or at least describe its main contents). Otherwise it's hard to understand what's happening in the following.

Reply:
Ok, I will add its content with short descriptions